src/Components/Footer.js

```diff
@@ -1,13 +1,143 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import styled from "styled-components";
 
-class componentName extends Component {
+class Footer extends Component {
   render() {
     return (
-      <div>
-        
+      <div className="Footer">
+        <FooterContainer>
+          <FooterBox>
+            <Top>
+              <Logo></Logo>
+              <TopInfoBox>
+                <li>블로그</li>
+                <li>이용약관</li>
+                <li>개인정보처리방침</li>
+                <li>운영정책</li>
+                <li>고객 문의</li>
+              </TopInfoBox>
+            </Top>
+            <Bottom>
+              <Info>
+                <li>상호명: 주식회사 앤스페이스</li>
+                <li>대표: 정수현</li>
+                <li>사업자등록번호: 230-81-03117</li>
+                <li>통신판매업신고번호: 2018-서울강남-04590</li>
+                <li>사업자등록정보 > </li>
+                <li>영업소재지: 서울특별시 강남구 삼성로 414</li>
+                <li>이메일: office@spacecloud.kr</li>
+                <li>대표전화: 1599-4312(평일 오후 2시 ~ 오후 6시)</li>
+                <li>온라인 1:1문의 바로가기(평일 오전 10시 ~ 오후6시)</li>
+                <li>
+                  *공간에 대한 문의사항은 해당 공간 호스트에서 직접
+                  문의해주세요.
+                </li>
+              </Info>
+              <FooterIcon>
+                <Icon></Icon>
+                <IconBlog></IconBlog>
+                <IconFacebook></IconFacebook>
+                <IconTwitter></IconTwitter>
+                <IconInsta></IconInsta>
```

Comment:
이거 다 self closing 태그로 바꿔주세요!
이렇게 태그만 필요하고 안에 내용이 필요 없을 때는 self closing으로 써주시면 더 보기 편합니다 :)
하나만 보여드릴게요!
```suggestion
                <Icon />
                <IconBlog></IconBlog>
                <IconFacebook></IconFacebook>
                <IconTwitter></IconTwitter>
                <IconInsta></IconInsta>
```

Reply:
네 수정했습니다~~~₩